localization/pt/builder/README.md

```diff
@@ -0,0 +1,155 @@
+---
+layout: pattern
+title: Builder
+folder: builder
+permalink: /patterns/builder/
+categories: Creational
+language: pt
+tags:
+ - Gang of Four
+---
```

Comment:
Please update the yaml frontmatter according to the updated requirements at https://github.com/iluwatar/java-design-patterns/wiki/01.-How-to-contribute. Basically you can just clone them from https://raw.githubusercontent.com/iluwatar/java-design-patterns/master/builder/README.md

Reply:
I updated the yaml frontmatter as requested. Thanks @iluwatar

Reply:
@iluwatar could you please check this PR. Thank you